app/scripts/components/analysis/results/index.tsx

```diff
@@ -130,10 +131,22 @@ export default function AnalysisResults() {
     aoi
   });
 
-  const availableDomain: [Date, Date] | null = useMemo(
-    () => (start && end ? [start, end] : null),
-    [start, end]
-  );
+  const availableDomain: [Date, Date] | null = useMemo(() => {
+    if (!start || !end) return null;
+    const onlySingleValues = requestStatus.every(
+      (rs) => rs.data?.timeseries.length === 1
+    );
+    if (!onlySingleValues) return [start, end];
+
+    // When all data only contain one value, we need to pad the domain to make sure the single value is shown in the center of the chart
+    const intervalsMs = requestStatus.map((rs) => {
+      const interval = rs.data?.timeDensity;
+      if (!interval) return 0;
+      return { day: 3600000, month: 2592000000, year: 31536000000 }[interval];
+    });
+    const maxInterval = max(intervalsMs) ?? 3600000;
+    return [new Date(+start - maxInterval), new Date(+end + maxInterval)];
```

Comment:
With these values, and depending on you selected date interval the dots will be positioned in different places.

Ex: https://deploy-preview-418--veda-ui.netlify.app/air-quality/analysis/results?start=2020-01-01T00%3A00%3A00.000Z&end=2020-01-04T00%3A00%3A00.000Z&datasetsLayers=no2-monthly&aoi=bgxlAgqvrGy~lEtrgdAgenjC%3Fj~hRw`yt%40
<img width="714" alt="image" src="https://user-images.githubusercontent.com/1090606/211368052-e3349736-8ca0-4498-a04d-613484033857.png">

It seems to me that it would be enough to find the date for the single value and add a day before and a day after. This would make the data point in the middle